requirements.txt

```diff
@@ -19,8 +19,9 @@ webargs==0.18.0
 slacker==0.8.6
 raven[flask]==5.8.1
 ujson==1.33
-pyelasticsearch==1.4
```

Comment:
hrmm. The load_legal_docs still uses pyelasticsearch. Want to leave it in or update those as well?